src/main/kotlin/com/github/julianhille/sqldelight/jscipherdriver/SqlJs.kt

```diff
@@ -0,0 +1,108 @@
+@file:JsModule("better-sqlite3-multiple-ciphers")
+@file:JsNonModule
+@file:Suppress("unused")
+
+package com.github.julianhille.sqldelight.drivers.jscipher
+import kotlin.js.Promise
+
+open external class Error
+
+open external class SqliteError: Error {
+  var name: String
+  var code: String
+  var message: String
+}
+
+external class DatabaseOptions {
+  var readonly: Boolean  = definedExternally // = false
+  val fileMustExist: Boolean  = definedExternally // = false
+  val timeout: Int  = definedExternally // = 5000
+  val verbose: Boolean  = definedExternally // = false
+  val nativeBindingPath: Boolean  = definedExternally // = false
```

Comment:
Double white spaces:

```suggestion
  var readonly: Boolean = definedExternally // = false
  val fileMustExist: Boolean = definedExternally // = false
  val timeout: Int = definedExternally // = 5000
  val verbose: Boolean = definedExternally // = false
  val nativeBindingPath: Boolean = definedExternally // = false
```